tests/Unit/PrestaShopBundle/EventListener/API/Context/ShopContextListenerTest.php

```diff
@@ -203,6 +203,7 @@ private function mockMultistoreFeature(bool $isUsed): MultistoreFeature|MockObje
     {
         $feature = $this->createMock(MultistoreFeature::class);
         $feature->method('isUsed')->willReturn($isUsed);
+        $feature->method('isActive')->willReturn($isUsed);
```

Comment:
I'm curious why you needed to update this, the ShopContext and listeners were not modified in this PR so I'm surprised the related tests need to

Reply:
Because in the ShopContextBuilder, we use `MultistoreFeature->isActive()`. (line 76 in ShopContextBuilder.php)
Maybe its wrong like this 🤔

Reply:
No it makes sense, but then it should have been integrated in the PR that added this in the ShopContextBuilder not in this one, right? 🤔